### DISCUSSION TITLE
Error in toml.so

### DISCUSSION THREAD

**Discussion by rxf4e1** [lumen-oss/rocks.nvim#239] [Q&A]
![image](https://github.com/nvim-neorocks/rocks.nvim/assets/66033296/a0baf281-cdfb-42c9-9ea1-4272fa468f8a)
I'm getting this error on every neovim startup, i checked the directory and toml.so is there, how do i solve it? I'm on nixos (unstable) and i have installed luajit and luajitPackages.luarocks

**Comment by mrcjkb** [comment]
Hey 👋 

It looks like toml.so has a link to libstdc++.so.6, which it can't find.

We're working on removing the toml dependency from rocks.nvim (see https://github.com/nvim-neorocks/rocks.nvim/issues/228).

**Reply by rxf4e1** [reply]
i managed to get it working by changing the default installation directory in .local/share/... to my own luarocks in $HOME/.luarocks and installed the dependencies manually.

**Reply by mrcjkb** [reply]
Thanks for sharing the solution :smile: 
The test derivation I use uses a `toml` package built with nix, so I never came across this :sweat_smile:
